File: sentimental-mario-less/mario.py
from cs50 import get_int
def main():
    while True:
        heights = get_int("Height: ")
        if heights not in range(1,9):
            continue
        else:
            print_heights(heights)
            break
def print_heights(x):
    for i in range(1,x+1):
        one = " "*(x-i)+("#"*i)
        print(one)
if __name__ == "__main__":
    main()

File: mario-less/mario.c
#include <stdio.h>
#include <cs50.h>
int main(void)
{   int x = 1;
    while (x==1)
    {
        int number = get_int("Height: ");
       if (8>=number && number >= 1)
       {
         for(int j = 0; j < number; j++)
    {
        for(int i = number-1 ; i-j > 0; i--)
    {
        printf(" ");
    }
    for (int k = 0; k<=j; k++){
        printf("#");
       }
    // printf("#");
    printf("  ");
    for (int k = 0; k<=j; k++){
        printf("#");
       }
    printf("\n");
    }
       }
       else
       {
        continue;
       }
       x = 0;
    }
}
